docker-compose.msls.yml

```diff
@@ -0,0 +1,77 @@
+version: '3'
+
+services:
+  msls:
+    build:
+      context: ./dev-local/msls
+      dockerfile: Dockerfile
+    command: msls
+    ports:
+      - "8080:8080"
+  db:
+    image: postgres
+    environment:
+            - POSTGRES_DB=bluebutton
+            - POSTGRES_PASSWORD=toor
+    ports:
+            - "5432:5432"
+  web:
+    build: . 
+    command: ./docker-compose/bluebutton_server_start.sh '${DB_MIGRATIONS}' '${SUPER_USER_NAME}' '${SUPER_USER_EMAIL}' '${SUPER_USER_PASSWORD}' '${BB20_ENABLE_REMOTE_DEBUG}' '${BB20_REMOTE_DEBUG_WAIT_ATTACH}'
+    environment:
+            - DJANGO_SETTINGS_MODULE=hhs_oauth_server.settings.dev
```

Comment:
For env vars, you can actually add them to a file and just use the file name here if that is easier for organization.

Reply:
Ex:
```
web:
  env_file:
    - web-variables.env
```
You can read more on it here: https://docs.docker.com/compose/environment-variables/

Reply:
We could.  I feel that some of the ENVs are better to have sourced vs. stored locally.  Deployment (Jenkins jobs) would likely need updated to work with this also. This might be a good improvements story for the backlog?

Reply:
I don't think it would have an impact on deployment. Could have @oragame or @njdister weigh in though. Basically the deployment jobs checkout the repo, and then run the docker-compose from there. You'd basically create the `.env` in the repo and check it in (since the values are already checked into the repo they are safe to be in there). It's just basically more organized to keep them in a separate file to really keep the docker-compose specific to the setup of the containers.

Reply:
Basically can we do: `AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION-us-east-1}` inside the `.env` file and it work the same?

Reply:
@nbragdon I'm liking this idea more after discussing with you in more detail. Will add this in :smile_cat:

Reply:
My .02 as far as .env's - I think in this particular case there is some repetition of environment vars on web and unittests in the dc file, so there could be some organization around that with .env files. I am thinking maybe a .common.env for the values used on both services, and then a specific one for web like .web.env - sounds like you have a plan, good call Nick!

Reply:
@nbragdon @oragame I just reworked the ENV configs. This tested out OK and looks cleaner.  Split the env files up by container and SLSx/MSLS. If having any issues, there is sometimes the need to docker-compose down/up to get changes to register.

@JFU-GIT I removed the shell arguments passed to the bluebutton_server_start.sh script. Hoping this doesn't break any functionality. In testing, the ENVs worked if sourced from a developer's .env or launching shell OK.